Exercises_Java/src/SchoolGrades.java

```diff
@@ -0,0 +1,43 @@
+// . EXERCISE TO PRACTICE IF-ELSE
+
+public class SchoolGrades {
+    private int age;
+
+    public int getAge() {
+        return age;
+    }
+
+    public void setAge(int age) {
+        this.age = age;
+    }
+
+
+    public SchoolGrades (int age) {
+        setAge(age);
```

Comment:
Aqui estas pasando una variable como parametro, puedes asignarlo directamente a con this.age a tu variable de instancia en lugar de llamar al metodo para que el lo haga, el setAge seria mas bien para asignar ese valor desde fuera, por ejemplo el main donde creas la instancia de la clase.